web_dashboard_tile/models/tile_tile.py

```diff
@@ -14,18 +14,56 @@
 from openerp.exceptions import ValidationError, except_orm
 
 
+def median(vals):
+    # https://docs.python.org/3/library/statistics.html#statistics.median
+    # TODO : refactor, using statistics.median when Odoo will be available
+    #  in Python 3.4
+    even = (0 if len(vals) % 2 else 1) + 1
+    half = (len(vals) - 1) / 2
+    return sum(sorted(vals)[half:half + even]) / float(even)
+
+FIELD_FUNCTIONS = {
+    'count': {
+        'name': 'Count',
+        'func': False,  # its hardcoded in _compute_data
+        'help': 'Number of records',
```

Comment:
I'm translating them afterwords, at the time of usage. But you're right, I should translate it here! Ty!